citesphere/src/main/java/edu/asu/diging/citesphere/web/user/SyncItemController.java

```diff
@@ -12,18 +12,34 @@
 import edu.asu.diging.citesphere.core.exceptions.GroupDoesNotExistException;
 import edu.asu.diging.citesphere.core.exceptions.ZoteroHttpStatusException;
 import edu.asu.diging.citesphere.core.service.ICitationManager;
+import edu.asu.diging.citesphere.core.service.giles.GilesUploadChecker;
 import edu.asu.diging.citesphere.user.IUser;
 
 @Controller
 public class SyncItemController {
     
     @Autowired
     private ICitationManager citationManager;
+    
+    @Autowired
+    private GilesUploadChecker uploadChecker;
 
     @RequestMapping(value="/auth/group/{zoteroGroupId}/items/{itemId}/sync")
     public String getItem(Authentication authentication, Model model, @PathVariable("zoteroGroupId") String zoteroGroupId, @PathVariable("itemId") String itemId, @RequestParam(required = false, value = "index") String index, @RequestParam(defaultValue = "1", required = false, value = "page") int page,@RequestParam(value="collectionId", required=false) String collectionId,
             @RequestParam(defaultValue = "title", required = false, value = "sortBy") String sortBy) throws GroupDoesNotExistException, CannotFindCitationException, ZoteroHttpStatusException {
         citationManager.updateCitationFromZotero((IUser)authentication.getPrincipal(), zoteroGroupId, itemId);
+        uploadChecker.checkUploadStatus(itemId, (IUser)authentication.getPrincipal());
```

Comment:
I actually don't think we want to check this here. Make it two different actions: update the item data from zotoro and check on the upload status of the giles upload.